static/src/js/components/TemporalDisplay/__tests__/TemporalSelectionDropdown.test.js

```diff
@@ -1,147 +1,210 @@
+
+    expect(startLabel).toBeInTheDocument()
   })
 
-  test('sets the end date correctly when an valid date is passed', () => {
-    const { enzymeWrapper } = setup()
-    const testObj = moment.utc('2012-01-01 12:00:00', 'YYYY-MM-DD HH:mm:ss', true)
+  test('sets the start date correctly when an valid date is passed', async () => {
+    const user = userEvent.setup()
+
+    setup()
+
+    await waitFor(async () => {
+      await user.click(screen.getByRole('button'))
+    })
+
+    const inputs = screen.getAllByRole('textbox')
+
+    const startTestObj = moment.utc('2012-01-01 12:00:00', true).format('YYYY-MM-DD HH:mm:ss')
+
+    fireEvent.change(inputs[0], { target: { value: startTestObj } })
 
-    enzymeWrapper.instance().setEndDate(testObj)
-    expect(enzymeWrapper.state().temporal.endDate).toEqual('2012-01-01T12:00:00.000Z')
+    expect(inputs[0].value).toBe(startTestObj)
+    expect(inputs[1].value).not.toBe(startTestObj)
   })
 
-  test('sets the state correctly with an invalid start date', () => {
-    const { enzymeWrapper } = setup()
-    const invalidDate = moment('2012-01-efss 12:00:00', 'YYYY-MM-DD HH:mm:ss', true)
-    const validStartDate = moment.utc('2012-01-01 12:00:00').toISOString()
-    const validEndDate = moment.utc('2012-01-02 12:00:00').toISOString()
+  test('sets the end date correctly when an valid date is passed', async () => {
+    const user = userEvent.setup()
+
+    setup()
+
+    await waitFor(async () => {
+      await user.click(screen.getByRole('button'))
+    })
+
+    const inputs = screen.getAllByRole('textbox')
 
-    enzymeWrapper.setState({ temporal: { startDate: validStartDate, endDate: validEndDate } })
-    enzymeWrapper.instance().setStartDate(invalidDate)
-    expect(enzymeWrapper.state().temporal.startDate).toBe('2012-01-efss 12:00:00')
-    expect(enzymeWrapper.state().temporal.endDate).toBe('2012-01-02T12:00:00.000Z')
+    const endTestObj = moment.utc('2015-01-01 12:00:00', true).format('YYYY-MM-DD HH:mm:ss')
+
+    fireEvent.change(inputs[1], { target: { value: endTestObj } })
+
+    expect(inputs[0].value).not.toBe(endTestObj)
+    expect(inputs[1].value).toBe(endTestObj)
   })
 
-  test('sets the state correctly with an invalid end date', () => {
-    const { enzymeWrapper } = setup()
-    const invalidDate = moment('2012-01-efss 12:00:00', 'YYYY-MM-DD HH:mm:ss', true)
-    const validStartDate = moment.utc('2012-01-01 12:00:00').toISOString()
-    const validEndDate = moment.utc('2012-01-02 12:00:00').toISOString()
+  test('sets the state correctly with an invalid start date', async () => {
+    const user = userEvent.setup()
+
+    setup()
+
+    await waitFor(async () => {
+      await user.click(screen.getByRole('button'))
+    })
+
+    const inputs = screen.getAllByRole('textbox')
+
+    const invalidDate = '2012-01-efss 12:00:00'
+    const validStartDate = moment.utc('2012-01-01 12:00:00').format('YYYY-MM-DD HH:mm:ss')
+    const validEndDate = moment.utc('2012-01-02 12:00:00')
```

Comment:
nitpick but, can you make the `validStartDate` and `validEnddate` format calls consistent here. Such as
```suggestion
    const validEndDate = moment.utc('2012-01-02 12:00:00').format('YYYY-MM-DD HH:mm:ss')
```
Then remove the `.toISOString()` call on line 126. The old test used toISOString for both not certain which one you need for this but, it seems like they should be the same

Similarly on line 129 I would think the assertion for  `validStartDate` and `validEndDate` would be the same